lib/kernel/src/prim_tty.erl

```diff
@@ -535,6 +536,8 @@ handle_request(State = #state{ xn = OrigXn, unicode = U }, {insert, Chars}) ->
      NewState};
 handle_request(State, beep) ->
     {<<7>>, State};
+handle_request(State, clear) ->
+    {<<"\e[H\e[2J">>, State};
```

Comment:
Use `tgetstr("clear")` to fetch this value so that if we happen across a shell that prefers to use another ANSI code for clear we will do so.